config/application.rb

```diff
@@ -23,6 +23,11 @@ class Application < Rails::Application
     # Initialize configuration defaults for originally generated Rails version.
     config.load_defaults 7.0
 
+    # Please, add to the `ignore` list any other `lib` subdirectories that do
+    # not contain `.rb` files, or that should not be reloaded or eager loaded.
+    # Common ones are `templates`, `generators`, or `middleware`, for example.
+    # config.autoload_lib(ignore: %w(assets tasks))
```

Comment:
IIUC this will add `lib` to our load paths, but as we are not autoloading any of the files located there I thought there was no need to have `lib` in our load path. We could add this line and ignore those files that we are requiring manually too but 🤷‍♂️ . What do you think?

Reply:
maybe we can do `config.autoload_lib(ignore: %w(assets tasks scraper)) `, so we autoload the lib folder like new apps do by default, but ignore the scraper files?

Reply:
Updated! [5f510d0](https://github.com/cedarcode/mi_carrera/pull/337/commits/5f510d038e7fe0e184e921fba764d7882af089ae)